.github/workflows/test.yml

```diff
@@ -91,3 +91,39 @@ jobs:
       - name: Run browser tests
         working-directory: ./acvm_js
         run: yarn test:browser
+
+  test-acvm_js-rust:
+    needs: [build-acvm-js-package]
+    name: Rust Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: acvm-js
+          path: ./result
+
+      - name: Set up test environment
+        uses: ./.github/actions/setup
+        with:
+          working-directory: ./acvm_js
+
+      - name: Setup Rust
```

Comment:
https://github.com/noir-lang/noir/blob/92af05a8f4219eb7e41348df8c5877b9d7ddb6db/.github/workflows/publish.yml#L61C7-L61C7

Can you change to using this action to install rust? We can also pass in the targets we want to install.